source/User_Guide/Marketing_Campaigns/custom_fields.md

```diff
@@ -65,9 +87,18 @@ Deleting a Custom Field
 1. Click **Confirm**.
 
 {% warning %}
-Deleting a custom field deletes all values for that field across your contact database. If you have any campaigns using the data in this field with a substitution tag, those values do NOT get replaced when you send the campaign. You cannot delete a custom field that is used in a segment. This deletion process may take several minutes--you continue to see the custom field on this page until the process has completed.
+Deleting a custom field deletes all values for that field across your contact database. If you have any campaigns using the data in this field with a substitution tag, those values do NOT get replaced when you send the campaign. You cannot delete a custom field that a segment is currently using. This deletion process may take several minutes--you continue to see the custom field on this page until the process has completed.
 {% endwarning %}
 
+{% anchor h2 %}
+Troubleshooting
+{% endanchor %}
+
+
+If a Custom Field value does not appear in the corresponding (Substitution Tag)[https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/editor.html#-Using-Substitution-Tags], make sure that there is a value for that custom field associated with the contact in your contact database. If there is no value for a particular custom field, a space will be substituted instead.
```

Comment:
This link is not rendering correctly - I have no idea. Make it relative, but I'm not sure that will fix it

Reply:
it's because I switched the [] and the (). Oops. But I still made it relative.